FordLabs/retroquest
# Update RetroQuest documentation

## Overview
- [x] Update RetroQuest documentation

## Testing Instructions
1) Ensure documentation makes sense and explains properly how to run the ui, api, e2e-tests, and setup the optional email server.
